.circleci/windows/msys2_prepare.bat

```diff
@@ -0,0 +1,6 @@
+@echo on
+:: Unset CircleCI default setting that prevents pulling Styrene from the public repo
+git config --global --unset url."ssh://git@github.com".insteadOf "https://github.com"
```

Comment:
CircleCI sets `git config --global url.ssh://git@github.com.insteadOf https://github.com` on the 'checkout' step and forgets to remove it. However it prevents to download from public repos without a key. So we need to revert this setting to be able to checkout Styrene (which has no release binary)